src/custom/pages/App/index.tsx

```diff
@@ -89,13 +90,19 @@ function createRedirectExternal(url: string) {
 
 const Loading = <LoadingWrapper>Loading...</LoadingWrapper>
 
+const getShowBannerState = (key: string) => {
+  return JSON.parse(localStorage.getItem(key) || 'true') // show banner by default
+}
+
 export default function App() {
+  const isVisible = useMemo(() => getShowBannerState(IS_SIDE_BANNER_VISIBLE_KEY), []) // empty array only calls on mount
```

Comment:
Better use a more clear name `isSideBannerVisible`

Reply:
Makes sense to me, committed your suggestion.